guide/tests/message_transformation_test.bal

```diff
@@ -0,0 +1,42 @@
+import ballerina/http;
+import ballerina/log;
+import ballerina/mime;
+import ballerina/io;
+import ballerina/mysql;
+import ballerina/test;
+
+boolean serviceStarted;
+
+function startService() {
```

Comment:
Start and Stop service functions are not required. Please check

Reply:
Done.